source/extensions/common/wasm/null/proxy_wasm_externs.h

```diff
@@ -0,0 +1,171 @@
+#pragma once
```

Comment:
Note: this file is named the same as the file in API, but it's completely different, which is quite confusing... especially because the `//source/extensions/common/wasm/null:null_plugin_hdr` target includes both of them.

Could we name it differently?

Reply:
wasm_api_imp.h done.